packages/x-components/src/design-system-deprecated/components/picture/default.scss

```diff
@@ -0,0 +1,43 @@
+.x-picture {
+  // border
+  border-radius: var(--x-size-border-radius-top-picture-default)
+    var(--x-size-border-radius-right-picture-default)
+    var(--x-size-border-radius-bottom-picture-default)
+    var(--x-size-border-radius-left-picture-default);
+
+  // color
+  background: var(--x-color-background-picture-default);
+
+  &__image {
+    // layout
+    display: block;
+
+    // others
+    object-fit: var(--x-object-fit-picture-default);
+    mix-blend-mode: var(--x-mix-blend-mode-picture-default);
+
+    &--fallback {
+      --x-object-fit-picture-default: var(--x-object-fit-picture-fallback-default) !important;
+      --x-mix-blend-mode-picture-default: var(--x-mix-blend-mode-picture-fallback-default) !important;
```

Comment:
Lots of doubts here: I kept this on the not deprecated stylesheet, but with the values defined in the new XDS (contain, normal, respectively)